client/packages/common/src/intl/locales/en/app.json

```diff
@@ -65,14 +65,18 @@
   "heading.password": "Password",
   "button.login": "Log in",
   "error.login": "Invalid username or password",
-  "error.site_auth_timeout": "Site authentication timed out",
-  "error.site_has_no_store": "Site has no store",
-  "error.site_incorrect_hardware_id": "Incorrect hardware ID",
-  "error.site_incorrect_password": "Incorrect password",
-  "error.site_name_not_found": "Site name not found",
-  "error.unable_to_save_settings": "Unable to save settings",
-  "error.unable_to_restart_server": "Unable to restart the server",
+  "error.site-auth-timeout": "Site authentication timed out, please try again later",
+  "error.site-has-no-store": "Site has no store",
+  "error.site-incorrect-hardware-id": "Incorrect hardware ID",
+  "error.site-incorrect-password": "Incorrect password",
+  "error.site-name-not-found": "Site name not found",
+  "error.integration-timeout-reached": "Central server integration timeout reached",
+  "error.invalid-url": "Invalid url",
+  "error.unable-to-save-settings": "Unable to save settings",
+  "error.unable-to-initialise": "Unable to initialise",
   "error.unknown-sync-error": "Unknown sync error",
+  "error.connection-error": "Unable to connect to server",
+  "error.site-uuid-is-being-changed": "Site uuid is different to recorded site uuid",
```

Comment:
This would only happen during initialisation or when url or site name is changed in sync settings, I think it's in the category of `user doesn't really know what sync is but knowing sync errors make our support live a lot easier` basket (like `Incorrect hardware id`, which was previously there). I just changed to `Site mismatch`: b250b11ca451f3e0506df3adbe5dbdfda4f8f8b7